flex-templates/java/regional_dlp_transform/src/main/java/com/google/cloud/blueprints/datawarehouse/Util.java

```diff
@@ -0,0 +1,227 @@
+import org.apache.beam.vendor.guava.v26_0_jre.com.google.common.collect.Lists;
+import org.apache.beam.vendor.guava.v26_0_jre.com.google.common.io.BaseEncoding;
+import org.joda.time.DateTimeZone;
+import org.joda.time.Instant;
+import org.joda.time.format.DateTimeFormat;
+import org.joda.time.format.DateTimeFormatter;
+import org.joda.time.format.DateTimeFormatterBuilder;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@SuppressWarnings("serial")
+public class Util {
+
+  public static final Logger LOG = LoggerFactory.getLogger(Util.class);
+
+  public enum DLPMethod {
+    INSPECT,
+    DEID,
+    REID
+  }
+
+  public enum FileType {
+    CSV,
+    AVRO,
+    JSON,
+    TXT
+  }
+
+  public static final Gson gson = new Gson();
+
+  private static final DateTimeFormatter BIGQUERY_TIMESTAMP_PRINTER;
+  public static final TupleTag<KV<String, String>> agentTranscriptTuple =
+      new TupleTag<KV<String, String>>() {};
+  public static final TupleTag<KV<String, String>> customerTranscriptTuple =
+      new TupleTag<KV<String, String>>() {};
+  public static final TupleTag<KV<String, String>> contentTag =
+      new TupleTag<KV<String, String>>() {};
+  public static final TupleTag<KV<String, ReadableFile>> headerTag =
+      new TupleTag<KV<String, ReadableFile>>() {};
+
+  public static final TupleTag<KV<String, TableRow>> inspectOrDeidSuccess =
+      new TupleTag<KV<String, TableRow>>() {};
+  public static final TupleTag<KV<String, TableRow>> inspectOrDeidFailure =
+      new TupleTag<KV<String, TableRow>>() {};
+
+  public static final TupleTag<KV<String, TableRow>> jobSuccess =
+      new TupleTag<KV<String, TableRow>>() {};
+  public static final TupleTag<KV<String, TableRow>> jobFailure =
+      new TupleTag<KV<String, TableRow>>() {};
+
+  public static final TupleTag<KV<String, InspectContentResponse>> inspectApiCallSuccess =
+      new TupleTag<KV<String, InspectContentResponse>>() {};
+  public static final TupleTag<KV<String, TableRow>> inspectApiCallError =
+      new TupleTag<KV<String, TableRow>>() {};
+
+  public static final String BQ_DLP_INSPECT_TABLE_NAME = String.valueOf("dlp_inspection_result");
+  public static final String BQ_ERROR_TABLE_NAME = String.valueOf("error_log");
+  public static final String BQ_REID_TABLE_EXT = String.valueOf("re_id");
+  public static final String BQ_DEID_TABLE_EXT = String.valueOf("de_id");
+
+  public static final DateTimeFormatter TIMESTAMP_FORMATTER =
+      DateTimeFormat.forPattern("yyyy-MM-dd HH:mm:ss.SSSSSS");
+
+  public static String checkHeaderName(String name) {
+    String checkedHeader = name.replaceAll("\\s", "_");
+    checkedHeader = checkedHeader.replaceAll("'", "");
+    checkedHeader = checkedHeader.replaceAll("/", "");
+    checkedHeader = checkedHeader.replaceAll("\\W", "");
+    LOG.debug("Name {} checkedHeader {}", name, checkedHeader);
+    return checkedHeader;
+  }
+
+  public static final Schema dlpInspectionSchema =
```

Comment:
does anything reference the inspection schema? I wonder if we can eliminate this code.